wbooks/tsconfig.json

```diff
@@ -58,14 +58,15 @@
     "paths": {
       "@app/*": ["src/app/*"],
       "@constants/*": ["src/constants/*"],
-      "@components" : ["src/app/components/*"],
-      "@hooks" : ["src/app/hooks/*"],
-      "@screens" : ["src/app/screens/*"],
-      "@config" : ["src/config/*"],
-      "@interfaces" : ["src/interfaces/*"],
-      "@redux" : ["src/redux/*"],
-      "@services" : ["src/services/*"],
-      "@utils" : ["src/utils/*"]
+      "@components/*" : ["src/app/components/*"],
```

Comment:
Nice catch!

Reply:
Yes, it took me several minutes looking elsewhere for the cause of the error hahaha...classic